.travis.yml

```diff
@@ -1,27 +1,36 @@
 language: php
 
+dist: xenial
+
+services:
+  - mysql
+  - postgresql
+
 env:
   global:
-    - COMPOSER_ROOT_VERSION="2.0.x-dev"
+    - COMPOSER_ROOT_VERSION="2.x-dev"
+    - SUBSITES=1
```

Comment:
We could just drop the condition from the `before_script` entry?

Reply:
done